### PR TITLE
add space travel story

### PR DESCRIPTION
**Pull request by berd7**

### CODE REVIEW

**Comment by berd7** [comment]
"Please review this, @casey-collab"

**Comment by berd7** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Hey! Me! Review this! @casey-collab

File: english/space-travel/space-travel.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Our planet has been ravaged. We are running low on resources therefore,
```
Comment (casey-collab) [inline english/space-travel/space-travel.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: english/space-travel/space-travel.md (hunk 1 above)
Comment (berd7) [reply english/space-travel/space-travel.md]: added space link to marshmellow.md, Please review this, @casey-collab.

**Comment by berd7** [comment]
just added space travel link to marshmellow.md so that it can be reached. Please review this, @casey-collab.
Thank you. Very useful class btw!

**Comment by casey-collab** [comment]
Looks great! :shell:

**Pull request merged**